MODULE.bazel.lock

```diff
@@ -1,5 +1,5 @@
 {
-  "lockFileVersion": 3,
```

Comment:
This regenned on `make generate` safe to inculde?

Reply:
CI doesn't seem bothered by it so I suspect it's ok.